### ISSUE TITLE
Is it possible to create an EPUB publication that conforms to both ISO/IEC 23761 and EPUB A11Y 1.1?

### ISSUE THREAD

**Issue by murata2makoto** [mattgarrish/epub-revision#2483]
ISO/IEC 23761 (or EPUB A11Y 1.0 by IDPF) uses the link element for specifying "conformsTo", but EPUB A11Y 1.1 uses the meta element for the same purpose.

Is it possible to have both in a single EPUB publication?  Does such an EPUB publication conform to both specifications?

```
   <link rel="dcterms:conformsTo" href="http://www.idpf.org/epub/a11y/accessibility-20170105.html#wcag-aa" />
    <meta 
        property="dcterms:conformsTo"
        id="conf">
        EPUB Accessibility 1.0 - WCAG 2.0 Level AA
    </meta>
```

**Comment by mattgarrish** [comment]
You can technically conform to both, but you'd need two separate conformsTo statements. For the 1.0/ISO specs, you can use either a meta or link:

```
<link rel="dcterms:conformsTo" href="http://www.idpf.org/epub/a11y/accessibility-20170105.html#wcag-aa" />
<meta property="dcterms:conformsTo">http://www.idpf.org/epub/a11y/accessibility-20170105.html#wcag-aa</meta>
```

If you want to claim conformance to both the 1.0 and 1.1 specifications -- which is only possible if you conform to WCAG 2.0, don't omit extended descriptions where necessary, and add the required accessibility summary for 1.0 -- you could add both:

```
<meta property="dcterms:conformsTo">http://www.idpf.org/epub/a11y/accessibility-20170105.html#wcag-aa</meta>
<meta property="dcterms:conformsTo">EPUB Accessibility 1.1 - WCAG 2.0 Level AA</meta>
```

Conformance to the 1.0 specification will always require an IDPF URL since we can't retroactively change it.

**Comment by clapierre** [comment]
Since conforming to WCAG 2.1 also means you conform to WCAG 2.0 I assume then you could state the following 
```
<meta property="dcterms:conformsTo">http://www.idpf.org/epub/a11y/accessibility-20170105.html#wcag-aa</meta>
<meta property="dcterms:conformsTo">EPUB Accessibility 1.1 - WCAG 2.1 Level AA</meta>
```
i.e. conforming to the original EPUB Accessibility 1.0 @ WCAG 2.0 and EPUB Accessibility 1.1 @ WCAG 2.1

Not sure if this is a good idea or not because some systems won't know what to do with multiple conformance statements and may only show the first or the last instance, where others may show both and I would think muddies the waters.

**Comment by murata2makoto** [comment]
@clapierre @mattgarrish 

Thanks.  

>Not sure if this is a good idea or not because some systems won't know what to do with multiple conformance statements and may only show the first or the last instance, where others may show both and I would think muddies the waters.

Do you have any advice for the Japanese publishing industry?  Now, Ace appears to support 1.0 only.  But 1.1 is a CR.  Should we start to use 1.1 now?  Or, should we wait for the completion of 1.1?

**Comment by mattgarrish** [comment]
> Now, Ace appears to support 1.0 only.

I believe it's been upgraded now, but I think it might only be the command-line version at this time. @danielweck would know more. The SMART tool now supports evaluating against both standards.

> Should we start to use 1.1 now? Or, should we wait for the completion of 1.1?

Depends on your risk tolerance. Given how late we are in the process, and that we've already had horizontal reviews, I don't see any major changes coming. But specifications can change until the very end. Usually we make things less strict at this stage, though, not more strict.

> Since conforming to WCAG 2.1 also means you conform to WCAG 2.0 I assume then you could state the following

Right, but I wouldn't think we'd want anyone conforming to 2.1 going back to make claims against 1.0. I could see someone who supports 1.0 maybe wanting to experiment with adding a 1.1 conformance. That's why I'd only suggest it if they're basically saying the same thing of WCAG 2.0 conformance. The EPUB parts are pretty minor in comparison, and I don't think most consumers of the metadata would know or understand the difference.

As for multiple statements, that's already allowed now so that alone shouldn't cause issues. The problem could be that a reading system will stop at the first one it finds that it understands. In that case, it would probably be best to list a 1.1 conformance statement before a 1.0. If it doesn't recognize 1.1, it will fall back to 1.0.

More practically, though, if there are problems with vendors not picking up the new 1.1 metadata field, then we need to talk to those vendors. Adding two statements to cover both versions should be a last resort. It's not something we should formally recommend anywhere.

**Comment by shiestyle** [comment]
@murata2makoto @mattgarrish
Japanese publishing industry will not use ver 1.0 and we will wait for that EPUB 3.3 (EPUB-A11Y 1.1) will be a recommendation.

**Comment by mattgarrish** [comment]
One thing we may want to emphasize after 1.1 becomes a recommendation is to migrate conformance statements even if publishers aren't ready to move to WCAG 2.1+.

So long as you didn't opt to exclude extended descriptions, content that conforms to the 1.0 specification can list the similar 1.1 conformance statement:

```
<meta property="dcterms:conformsTo">EPUB Accessibility 1.1 - WCAG 2.0 Level AA</meta>
```

It's better to begin using this new conformance pattern than to persist the old IDPF URL until publishers are ready for WCAG 2.1.

**Comment by murata2makoto** [comment]
It appears that we need a PAS submission of EPUB A11Y 1.1 to ISO/IEC JTC1, immediately after 1.1 becomes a REC.  I can then start another JIS committee for updating the JIS version.

**Comment by mattgarrish** [comment]
Are you okay with closing this issue now @murata2makoto ?

**Comment by mattgarrish** [comment]
Closing this as it's morphed into a W3C process discussion that's now on the mailing list.
